prov/verbs/src/ep_rdm/verbs_cq_ep_rdm.c

```diff
@@ -216,10 +216,12 @@ fi_ibv_rdm_cq_readerr(struct fid_cq *cq_fid, struct fi_cq_err_entry *entry,
 						FI_LOG_DEBUG);
 			util_buf_release(fi_ibv_rdm_request_pool, err_request);
 		} else {
-			err_request->state.eager = FI_IBV_STATE_EAGER_READY_TO_FREE;
+			err_request->state.eager = FI_IBV_STATE_EAGER_READY_TO_FREE;			
```

Comment:
nit: is there extra space at the end of the line?

Reply:
Thanks! I've fixed this